lib/client/dio.dart

```diff
@@ -23,7 +23,7 @@ Dio httpClient(
 
   final options = BaseOptions(
     baseUrl: 'https://api.weatherapi.com/v1',
-    queryParameters: {'key': '065505914dc344ad95c121519231307'}, // DO NOT commit this key
+    queryParameters: {'key': '<your-api-key>'}, // DO NOT commit this key
```

Comment:
My concern is that this key is still visible in the commit history. Maybe let's just delete this permanently and redo this again, ok?

I'm sorry, it's for your own security

Reply:
Hello! Thanks, is not problem, I already invalided the api-key. You can use this if you like this PR.